src/lib/_navigation.scss

```diff
@@ -22,6 +22,7 @@ $navigationBtnWidth: 50px;
     transition-property: opacity;
     white-space: nowrap;
     width: $navigationBtnWidth;
+    z-index: 1;
```

Comment:
What's the reason for this change?

Reply:
Hm, can we take a look at this tomorrow? It doesn't seem like the buttons should need to change to support the navigation bar.